dev-lang/zig/zig-9999.ebuild

```diff
@@ -23,14 +23,18 @@ RESTRICT="!test? ( test )"
 
 BUILD_DIR="${S}/build"
```

Comment:
for all commit messages: please no version before the colon (app-misc/foo: blah, but not app-misc/foo-9999: blah. you can do app-misc/foo: blah in 9999 if you want).